src/main/java/oop/entertainment/Entertainment.java

```diff
@@ -0,0 +1,38 @@
+package oop.entertainment;
+/*
+ * creates a class called entertainment that cannot have an instance only described as that. 
+ */
+public class Entertainment {
+    /* 
+     * sets up the class attribute "title," as all entertainment has a title. 
+     */ 
+    private String title;
+    /* 
+     * uses the getter method for title - accesses the private specific variable title. 
+     */
+    public String getTitle() {
+        return this.title;
+    }
+    /*
+     * uses the setter method - sets a new instance of title. 
+     */
+    public void setTitle(String title) {
+        this.title = title;
+    }
+    /*
+     * sets up the class attribute "pasttime," as all entertainment is a pasttime. 
+     */
+    private String pasttime;
```

Comment:
this is a fine location for this data member, but generally we list the data members at the top and let the setters / getters follow for readability purposes.